src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandler.java

```diff
@@ -88,6 +91,33 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
         return preSubmitCallbackResponse;
     }
 
+    private void checkSscs2Confidentiality(PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse, SscsCaseData sscsCaseData) {
```

Comment:
This is the same method in OtherPartyDataUtil class, isn't it? If so, We can use the util method and remove this one

Reply:
no its slightly different for DwpUploadResponse as the edited dwp documents are involved in the logic